### ISSUE TITLE
Install broken with gcc 6?

### ISSUE THREAD

**Issue by josephwb** [FePhyFoFum/phyx#32]
Updated gcc on my mac to v6.1.0 and now `sudo make install` does not work:
```
Frinkatron-TNG:src josephwb$ sudo make install
install -m 0755 px* /usr/bin
install: /usr/bin/pxaatocdn: Operation not permitted
make: *** [install] Error 71
```
A simple work around is to simply change:
```
install -m 0755 px* /usr/bin
```
to:
```
cp px* /usr/bin
```
but that isn't very satisfying. Perhaps @jfwalker should try doing this on his system to rule out some weird setup of my own.

It doesn't seem that debian yet supports gccv6.1.0 (I am actually using gccv4.9.2?). We should try some other flavour of linux (ubuntu) to see if it is indeed gcc to blame. I will reinstall gccv5 on my mac to check things there.

**Comment by josephwb** [comment]
Hmm. gcc-5 (on mac) also has this problem...

**Comment by josephwb** [comment]
_and_ gccv4.8. I _think_ this was working before...

**Comment by josephwb** [comment]
Well, crap. This is apple's fault, not gcc.

Apple made a change where (I am not shitting you) a user, even using `sudo`, cannot copy something into `/usr/bin`.

So, looks like we have to find a better default install location. Ideas?

**Comment by josephwb** [comment]
Ok, this is fixed by c918a068fc71bba6c3996f6d68079ffd09d52249 by changing default install location to `/usr/local/bin`

On linux, installing here requires `sudo`, but on mac does not. Blech.
